src/handlers/startGame.js

```diff
@@ -2,16 +2,23 @@
 import { setNextQuestion } from "../handlers/setNextQuestion.js";
 import { data } from '../data.js';
 import { quizData } from '../data.js';
+import { renderScore } from "./render-score.js";
 
 export function startGame() {
```

Comment:
So this is now a "start or restart" function, right? Might be good to add that as a comment above it.